flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -170,6 +171,8 @@ public class YarnClusterDescriptor implements ClusterDescriptor<ApplicationId> {
 
     private YarnConfigOptions.UserJarInclusion userJarInclusion;
 
+    private static final long BYTES_IN_MB = 1024 * 1024;
+
```

Comment:
```suggestion
```

It can be deleted.

Reply:
Thanks for your suggestion, I will modify the code.